### PR TITLE
meta

### PR DESCRIPTION
**Pull request by Rulaazz**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,24 +2,37 @@
...
+    <meta name="description" content="An engaging quiz app promoting peace and understanding through fun questions about diverse cultures. Celebrate diversity and unity by exploring the richness of global traditions, people, and communities." />
+    <meta name="keywords" content="quiz app, cultural diversity, religious diversity, famous people, trivia, Levantini, peace, understanding, global traditions, unity" />
+
+    
+    <meta name="theme-color" content="#ffff" /> 
+
+    <!-- Open Graph Tags -->
+    <meta property="og:title" content="Levantini" />
+    <meta property="og:description" content="An engaging quiz app promoting peace and understanding through fun questions about diverse cultures. Celebrate diversity and unity by exploring the richness of global traditions, people, and communities." />
+    <meta property="og:image" content="/src/logo.png" />
```
Comment (pini85) [inline index.html]: meta tags are part of the static HTML, crawlers cannot access images located in src, as this folder is part of the JavaScript bundling process and not directly accessible from the root URL.
Like so:
public/images/logo.png

File: index.html [hunk 2]
```diff
@@ -2,24 +2,37 @@
...
+<link
+  href="https://fonts.googleapis.com/css2?family=ABeeZee:ital@0;1&family=Redressed&family=Rubik:ital,wght@0,300;0,400;0,500;0,700;0,900;1,300;1,400;1,500;1,700;1,900&display=swap"
+  rel="stylesheet"
+/>
+    <title>Levantini</title>
+    <meta name="description" content="An engaging quiz app promoting peace and understanding through fun questions about diverse cultures. Celebrate diversity and unity by exploring the richness of global traditions, people, and communities." />
+    <meta name="keywords" content="quiz app, cultural diversity, religious diversity, famous people, trivia, Levantini, peace, understanding, global traditions, unity" />
+
+    
+    <meta name="theme-color" content="#ffff" />
```
Comment (pini85) [inline index.html]: That is not a valid color. white color is #ffffff

File: index.html (hunk 1 above)
Comment (pini85) [inline index.html]: Best practice for open graph images for social sharing is 1200x630 pixels. Lets perhaps take a small screen shot from figma of our website and put that as the image

File: index.html [hunk 3]
```diff
@@ -2,24 +2,37 @@
...
+    
+    <meta name="theme-color" content="#ffff" /> 
+
+    <!-- Open Graph Tags -->
+    <meta property="og:title" content="Levantini" />
+    <meta property="og:description" content="An engaging quiz app promoting peace and understanding through fun questions about diverse cultures. Celebrate diversity and unity by exploring the richness of global traditions, people, and communities." />
+    <meta property="og:image" content="/src/logo.png" /> 
+    <meta property="og:url" content="https://levantini.netlify.app/" /> 
+    <meta property="og:type" content="website" />
+
```
Comment (pini85) [inline index.html:25]: we are missing the twitter one

File: index.html [hunk 4]
```diff
@@ -2,24 +2,37 @@
...
+    <!-- Favicon -->
+    <link rel="icon" href="/src/logo.png" /> 
   </head>
   <body>
     <div id="root"></div>
+    <h1>Hello</h1>
```
Comment (pini85) [inline index.html]: please remove this

**Review by pini85 — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Rulaazz) [reply index.html]: done

File: index.html (hunk 4 above)
Comment (Rulaazz) [reply index.html]: done, it was to check somthing

File: index.html (hunk 1 above)
Comment (Rulaazz) [reply index.html]: ok

File: index.html (hunk 3 above)
Comment (Rulaazz) [reply index.html:25]: I added

**Pull request closed without merging**